app/views/shared/_search.html.erb

```diff
@@ -25,7 +27,8 @@
   </header>
 <% elsif %w(search search_references exchange_rates pages errors feedback).exclude?(controller_name) && action_name != 'quota_search' %>
   <header>
-    <h1 class="govuk-heading-l"><a href="<%= sections_path %>"><%= default_heading %></a></h1>
+    <span class="govuk-caption-m"><%= trade_tariff_heading %></span>
+    <h1 class="govuk-heading-l"><a href="<%= sections_path %>"><%= default_heading(section: @section) %></a></h1>
```

Comment:
This could use rails' `link_to` ? Then it will handle param escaping in the sections path